sql/src/test/java/io/crate/integrationtests/AnyIntegrationTest.java

```diff
@@ -105,6 +106,7 @@ public void testAnyOnArrayLiteral() throws Exception {
     }
 
     @Test
+    @UseSemiJoins(0) // Execute explicitly both with semi_joins enabled and disabled
```

Comment:
0 means diabled. Still it executes with both?

Reply:
yes because we use this special method `exexuteWith`